tasks/collect-collectors-data/collect-collectors-data.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: collect-collectors-data
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: Tekton task to collect the information related to collectors
+  params:
+    - name: previousRelease
+      type: string
+      description: The namespaced name of the previous Release
+    - name: release
+      type: string
+      description: The namespaced name of the Release
+    - name: releasePlan
+      type: string
+      description: The namespaced name of the ReleasePlan
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: previousRelease
+      type: string
+      description: The relative path in the workspace to the stored previous release json
+    - name: release
+      type: string
+      description: The relative path in the workspace to the stored release json
+    - name: releasePlan
+      type: string
+      description: The relative path in the workspace to the stored releasePlan json
+    - name: releasePlanAdmission
+      type: string
+      description: The relative path in the workspace to the stored releasePlanAdmission json
+    - name: resultsDir
+      type: string
+      description: The relative path in the workspace to the results directory
+  steps:
+    - name: collect-collectors-data
+      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      env:
+        - name: "PREVIOUS_RELEASE"
+          value: '$(params.previousRelease)'
+        - name: "RELEASE"
+          value: '$(params.release)'
+        - name: "RELEASE_PLAN"
+          value: '$(params.releasePlan)'
+        - name: "RELEASE_PLAN_ADMISSION"
+          value: '$(params.releasePlanAdmission)'
+      script: |
+        #!/usr/bin/env bash
+        set -eo pipefail
```

Comment:
We want set -x here. Same reason we added it to collect-data. It makes debugging way easier

Reply:
Collect-data doesn't have it there.

Reply:
Oh, I thought it did. Not sure how collect-data puts it all in stdout now then